### DISCUSSION TITLE
Possibility about apply this library in Matlab

### DISCUSSION THREAD

**Discussion by YKK1024** [ros-industrial/abb_libegm#139] [Q&A] [answered]
Hello, could you tell me if I can use Matlab based on this egm library and socket to control the virtual robot in RobotStudio, especially for GoFa15000.

**Comment by gavanderhoorn** [comment, marked as answer]
There shouldn't be anything in `abb_libegm` which prevents you from using it with Matlab, but there is also no explicit support for Matlab.

So you'd be on your own when it comes to integration.

There might already be Matlab EGM integrations -- perhaps you could try to find one. It could save you a lot of work.
